windows/utils/win_get_bitlocker_volume.yml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+# Get the volumes encrypted by BitLocker Device Encryption
+# Return:
+#   bitlocker_volume_list: the list of encrypted volume drives
+
+- name: "Initialize the variables"
+  ansible.builtin.set_fact:
+    bitlocker_volume_list: []
+
+- name: "Get BitLocker encrypted volumes"
+  include_tasks: win_execute_cmd.yml
+  vars:
+    win_powershell_cmd: "(Get-BitLockerVolume | Where-Object { $_.EncryptionPercentage -GT 0 }).MountPoint"
+
+- name: "Set bitlocker volume list"
```

Comment:
'bitlocker' -> can be changed to the same format as others

Reply:
fixed.